### PR TITLE
Build

### PR DESCRIPTION
**Pull request by horta**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
```
Comment (ocefpaf) [inline recipe/meta.yaml:15]: Why did you need a build number bump without any changes to the package? Build number 0 is the same as build number 1 here. It only wastes space on anaconda.org.

Comment (horta) [reply recipe/meta.yaml:15]: Looks like I didn't have to...
